tests.py

```diff
@@ -110,6 +110,11 @@ def isSocketClosed(self, a):
             msg = a.recv(1)
         except socket.timeout:
             is_closed = False
+        except socket.error as e:
```

Comment:
OK. Probably I close the connection in "too hard way" (by RST instead of FIN-ACK).
If only FIN-ACK is allowed, then we shouldn't handle this exception.